mcs/class/System/System.Net.Sockets/Socket.cs

```diff
@@ -999,7 +999,7 @@ public static void CancelConnectAsync (SocketAsyncEventArgs e)
 			}
 		});
 
-		public IAsyncResult BeginConnect (string host, int port, AsyncCallback requestCallback, object state)
+		public IAsyncResult BeginConnect (string host, int port, AsyncCallback callback, object state)
 		{
```

Comment:
Verified this; the new name matches the one in CoreFx.  https://github.com/dotnet/corefx/blob/8d6320189d6df167680a203422a2235d9befb9f4/src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs#L2069